ee/vellum_cli/image_push.py

```diff
@@ -72,9 +73,29 @@ def image_push_command(image: str, tags: Optional[List[str]] = None) -> None:
             except Exception:
                 continue
 
-    logger.info("Updating Vellum metadata and validating image works in our system...")
-    image_details = docker_client.api.inspect_image(image)
-    sha = image_details["Id"]
+    result = subprocess.run(
+        ["docker", "inspect", "--format='{{index .RepoDigests 0}}'", image],
+        stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE,
+    )
```

Comment:
[oos] would be nice to add tests - down to defer if hard to mock out the response to the docker inspect command

Reply:
Ya I can followup with this and ya I defered on it since basically every line here needs to get mocked out.